.devops/azure-templates/setup-project.yml

```diff
@@ -0,0 +1,29 @@
+# Azure DevOps pipeline template used to setup the Node project:
```

Comment:
@aomegax we can remove this file and refer to [pagopa/azure-pipeline-templates]( https://github.com/pagopa/pagopa-mock-ec/blob/8a941cae3c62d41ea3cefe8126900dbe2ee57428/.devops/code-review-pipelines.yml#L18) directly in our pipelines. It was imported locally for io-pay project that managed FE and Be in the same repo. Isn't our case.